pages/register.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div class="bg-[#210309] flex justify-center items-center min-h-screen">
+    <FormKit
+      type="form"
+      v-model="data"
+      @submit="handleSubmit"
+      form-class="
+    bg-white rounded p-8 max-w-sm mt-20
+    "
+      :submit-attrs="{
+        'input-class':
+          ' ml-[240px] bg-[#F1023D] mt-3 no-underline text-white font-semibold py-2 px-4 rounded-full hover:text-white hover:bg-black shadow cursor-pointer outline-none focus:outline-none inline-block',
+      }"
+    >
+      <div class="space-y-4">
+        <img src="" alt="" class="w-40 justify-center align-middle mx-auto" />
+
+        <FormKit
+          type="text"
+          label="Email"
+          name="email"
+          validation="required|email"
+          :classes="{
+            label: classes.label,
+            input: classes.input,
+            error: 'text-red-500',
+            help: 'text-gray-500',
+          }"
+        />
+        <FormKit
+          type="password"
+          label="Password"
+          name="password"
+          validation="required|min:6"
+          :classes="{
+            label: classes.label,
+            input: classes.input,
+            error: 'text-red-500',
+            help: 'text-gray-500',
+          }"
+        />
+        <FormKit
+          type="text"
+          label="Username"
+          name="username"
+          validation="required|min:6"
+          help="Use only letters, numbers, underscores and periods"
+          :classes="{
+            label: classes.label,
+            input: classes.input,
+            error: 'text-red-500',
+            help: 'text-gray-500',
+          }"
+        />
+
+        <FormKit
+          type="select"
+          label="Dancing in"
+          name="dancing_in"
+          placeholder="City"
+          :options="['Pizza', 'Ice Cream', 'Burger']"
```

Comment:
Here we need to build a custom input, that uses Google Places API to autocomplete the city